Add a small note about the ROS2 humble branch. (#24)
open-rmf/fleet_adapter_template: README.md

# fleet_adapter_template

> > Note: If you are on ros2 humble and below take a look at the [humble](https://github.com/open-rmf/fleet_adapter_template/tree/humble) branch.

Reviewer: ```suggestion
> Note: If you are on using Open-RMF binaries binaries from ROS 2 Humble or an older distribution, switch to the [humble](https://github.com/open-rmf/fleet_adapter_template/tree/humble) branch.
```

> > Note: If you are using Open-RMF binaries binaries from ROS 2 Humble or an older distribution, switch to the [humble](https://github.com/open-rmf/fleet_adapter_template/tree/humble) branch.

Reviewer: ```suggestion
> Note: If you are using Open-RMF binaries from ROS 2 Humble or an older distribution, switch to the [humble](https://github.com/open-rmf/fleet_adapter_template/tree/humble) branch.
```

> > Note: If you are on using Open-RMF binaries binaries from ROS 2 Humble or an older distribution, switch to the [humble](https://github.com/open-rmf/fleet_adapter_template/tree/humble) branch.

Reviewer: ```suggestion
> Note: If you are using Open-RMF binaries binaries from ROS 2 Humble or an older distribution, switch to the [humble](https://github.com/open-rmf/fleet_adapter_template/tree/humble) branch.
```
